server/broadcast.go

```diff
@@ -997,13 +1001,13 @@ func shouldRefreshSession(sess *BroadcastSession) (bool, error) {
 	return false, nil
 }
 
-func newTranscodeEvent(mid core.ManifestID, seg *stream.HLSSegment, startTime time.Time, success bool, attempts []data.TranscodeAttemptInfo) (key string, event *data.TranscodeEvent) {
+func newTranscodeEvent(mid core.ManifestID, streamID string, seg *stream.HLSSegment, startTime time.Time, success bool, attempts []data.TranscodeAttemptInfo) (key string, event *data.TranscodeEvent) {
```

Comment:
@jailuthra @yondonfu @kyriediculous what do you think about using named return values? We haven't used them in before, and, for me, they make code harder to read. Personally I would prefer not to use them.

Reply:
Makes sense to stick to a single convention throughout the repo

Reply:
I particularly like them when the return type itself doesn't say much about what the returned value means, like a `string` in this case doesn't say that it's the message (event) routing key (and I don't think it's worth creating a type alias only for this).

But no strong preference here and I wasn't aware this is not used across `go-livepeer`, so I think it makes sense to avoid for consistency. Fixed.

Reply:
Comment on named return values:

I do think that named return values can be useful in certain scenarios where the named return value can clarify what is being returned, but...I also think in many (not all!) of those scenarios the need to use a named return value may be a code smell indicating there may be an opportunity to simplify the return values for a function. For example, in this case, I wonder if a separate `newEventKey(core.ManifestID, string) string` function that returns a key would have helped since it feels like the key definition need not be tied with the creation of the `data.TranscodeEvent`.

Reply:
Good point! I didn't want to create too many helpers here since that doesn't seem to be the practice in `go-livepeer` either, but if you're OK with that I do prefer that version :)
On it